### ISSUE TITLE
movie trailers and movie/show info

### ISSUE THREAD

**Issue by xsmashx88x** [tknorris/salts#148]
any chance you can add a option for this to show on the menu option, cause the only way i can get show or movie info is via a remote app on my phone.. would be nice to have it show on salts to view trailers and info

**Comment by tknorris** [comment]
It's already there is there's room in the menu. If you dont see it, then you need to turn some options off that trigger menu items like Download Source.

**Comment by xsmashx88x** [comment]
oh okay, will do that cause i didnt see it and was wondering. thanks for the fast reply

**Comment by Grossdm** [comment]
Please provide more details on how to do this.
Which item's menu might have these options, and specifically how can I enable or disable them?

Thank you so much for your excellent add-on!

**Comment by tknorris** [comment]
The following settings add additional context menus:
1. Download - Turn off in addon settings.
2. Setting a subscription list - There's no easy way to remove one once it's set, so you'd have to edit settings.xml and remove the value for sub_slug
3. Subtitles - Turn off in addon settings

**Comment by xsmashx88x** [comment]
also hitting "i" bring it up :) so try what norris said if not i key or info key will bring up the option

**Comment by tknorris** [comment]
Yep, "I" works if you have that button. Lots of remotes these days have limited buttons.

**Comment by Grossdm** [comment]
@tknorris 
Okay, perhaps I was thinking that there was more content (metadata and trailers, specifically) available with the described changes.
I am familiar with the i key for getting information on the highlighted (selected?) item, but everywhere that I can think of has "... information" on the context menu.  That would be for a TV show, an episode, etc.
I've been trying another add-on (no, I'm not monogamous), and it has trailers for the same item in SALTS much more often.  The impression I had was that there was a method of increasing the amount of these in SALTS.
Regardless, *thank you* very much for your fast and detailed response!

BTW, what was the deal with version 2.0.4? I came and left fast, jumping to 2.0.5.  I was curious, so I looked at the changelogs but they were identical except the 2.0.4 was now 2.0.5.
Interesting...  :wink:
